src/Illuminate/Database/Schema/Grammars/Grammar.php

```diff
@@ -96,6 +96,18 @@ public function compileChange(Blueprint $blueprint, Fluent $command, Connection
         throw new LogicException('This database driver does not support modifying columns.');
     }
 
+    /**
+     * Compile a primary key command.
+     *
+     * @param  \Illuminate\Database\Schema\Blueprint  $blueprint
+     * @param  \Illuminate\Support\Fluent  $command
+     * @return string
+     */
+    public function compilePrimary(Blueprint $blueprint, Fluent $command)
+    {
+        throw new RuntimeException('This database driver does not support adding primary key.');
```

Comment:
Should this be pluralized? i.e. "primary keys”

Reply:
Thanks.